src/test/java/divaeva/hw/api/store/CreateStoreTest.java

```diff
@@ -0,0 +1,36 @@
+package divaeva.hw.api.store;
+
+import divaeva.hw.api.pet.PetTestBase;
+import dto.PetDTO;
+import dto.StoreDTO;
+import io.restassured.response.Response;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import rest.client.RestClient;
+
+public class CreateStoreTest extends StoreTestBase {
+    private RestClient<StoreDTO> restClient = new RestClient<>(BASE_SERVER_URL_STORE);
```

Comment:
I'm not sure that its a good idea for generic here. What if you need to work with different entities in one test?

Reply:
fixed